src/main/python/generate.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I suggest to keep this .sh script in `src/main/resources`, as it's follows the standard project layout, and most probably this script will also include generating gRPC/Protobuf for other languages.

Reply:
And it's also good idea to add a small comment here, following https://google.github.io/styleguide/shell.xml#File_Header